### PR TITLE
#159236204-Allow comparison of training members

### PR DESCRIPTION
**Pull request by solnsubuga**
#### What does this PR do?
- Allows comparison of two users who might be training together based on weight entry data.
#### Description of Task to be completed?
- Allow a trainer to select two or more users and compare them.
#### How should this be manually tested?
-  Checkout the branch `ft-compare-members-159236204`
- Run the command `sudo python manage.py bower_install --allow-root `
- Run `cd ./extras/dummy_generator`
- To add some dummy members  run `python generator.py users 10`
- Add some weights `python generator.py weight 10`
- Run the application `python manage.py runserver`
- Navigate to gym members page
- Select two or more members from the members' list
- Click the compare members button
#### What are the relevant pivotal tracker stories?
- [#159236204](https://www.pivotaltracker.com/story/show/159236204)
#### Screenshots (if appropriate)
screen shot 2018-08-22 at 14 24 03
screen shot 2018-08-22 at 16 04 43

### CODE REVIEW

File: wger/weight/views.py [hunk 1]
```diff
@@ -190,6 +191,29 @@ def get_weight_data(request, username=None):
     return Response(chart_data)
 
 
+@api_view(['GET'])
+def get_user_weight_data(request):
+    q = request.GET.get('usernames', '')
```
Comment (wasswa-derick) [inline wger/weight/views.py]: Variable name q could be made more descriptive.

File: wger/gym/templates/gym/member_list.html [hunk 2]
```diff
@@ -139,7 +139,108 @@ <h5 class="modal-title" id="delete-modal-title" style="font-size: 140%; color: #
...
+    function compareMembers(event){
+        if(selectedUsers.length === 0){
+            $("#error-no-members-modal").modal("show");
+            return
+        }
+        var usernames = selectedUsers.join(',');
+        var colors = [
+        "#FFB81D","#AA0050","#0033a1","#665EC7","#7C868D","#00a1e0","#0a2240",
+        "#FF681D","#00A1B0","#B150C5",
+
```
Comment (wasswa-derick) [inline wger/gym/templates/gym/member_list.html]: Blank line not useful here.

File: wger/weight/views.py (hunk 1 above)
Comment (solnsubuga) [reply wger/weight/views.py]: I have changed it to usernames_param.

File: wger/gym/templates/gym/member_list.html (hunk 2 above)
Comment (solnsubuga) [reply wger/gym/templates/gym/member_list.html]: Removed it.

**Review by wasswa-derick — APPROVED** [review]
LGTM

File: wger/gym/templates/gym/member_list.html [hunk 3]
```diff
@@ -139,7 +139,107 @@ <h5 class="modal-title" id="delete-modal-title" style="font-size: 140%; color: #
...
+        </div>
+    </div>
+</div>
+
 <script>
+    var selectedUsers = [];
+    function selectUser(event,id){
+        if(event.checked == true){
+            selectedUsers.push(id);
+        }else{
```
Comment (wasswa-derick) [inline wger/gym/templates/gym/member_list.html]: Add spaces between the operators and the keyword 'else'.

**Pull request merged**